packages/frontend/src/help/analysis/stochastic-mass-action.mdx

```diff
@@ -0,0 +1,12 @@
+import { A } from "@solidjs/router";
+
+#### User inputs
+
+<dl>
+  <dt>Initial value: $\mathbb{R}_{\geqslant0}$</dt>
+  <dd>The initial value of the concentration.</dd>
+  <dt>Rate: $\mathbb{R}_{\geqslant0}$</dt>
+  <dd>The rate coefficient ($r$) of the reaction.</dd>
+  <dt>Duration: $\mathbb{R}_{\geqslant0}$</dt>
+  <dd>The total duration of the simulation in units of time.</dd>
```

Comment:
I noticed that you removed spurious periods from other definition lists, but a few periods remain here.

(This is pedantry but I appreciate this level of pedantry coming from you!)